### PR TITLE
Fixed some phpdocs in lib/*

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by fballiano — APPROVED** [review]

File: lib/Mage/Backup/Filesystem.php [hunk 1]
```diff
@@ -87,13 +87,14 @@ public function rollback()
         $rollbackWorker->run();
 
         $this->_lastOperationSucceed = true;
+        return $this->_lastOperationSucceed;
```
Comment (kiatng) [inline lib/Mage/Backup/Filesystem.php:90]: Should lines 76 and 97 change to

```php 
     * @return true // from bool
```

File: lib/Mage/Backup/Filesystem.php [hunk 2]
```diff
@@ -87,13 +87,14 @@ public function rollback()
...
     }
 
     /**
      * Implementation Create Backup functionality for Filesystem
      *
      * @throws Mage_Exception
-     * @return boolean
+     * @return bool
```
Comment (kiatng) [inline lib/Mage/Backup/Filesystem.php:97]: It can only return true?

```suggestion
     * @return true
```

File: lib/Mage/System/Ftp.php [hunk 3]
```diff
@@ -51,8 +51,8 @@ protected function checkConnected()
     /**
      * ftp_mkdir wrapper
      *
-     * @param stin$name
-     * @return unknown_type
+     * @param string $name
+     * @return string
```
Comment (kiatng) [inline lib/Mage/System/Ftp.php:55]: https://www.php.net/manual/en/function.ftp-mkdir.php
> Returns the newly created directory name on success or false on error. 

```suggestion
    * @return string|false
```

File: lib/Mage/System/Ftp.php [hunk 4]
```diff
@@ -211,7 +211,7 @@ public function getcwd()
      * ftp_raw wrapper
      *
      * @param string $cmd
-     * @return mixed
+     * @return array
```
Comment (kiatng) [inline lib/Mage/System/Ftp.php:214]: https://www.php.net/manual/en/function.ftp-raw.php
> Returns the server's response as an array of strings, or null on failure. No parsing is performed on the response string, nor does ftp_raw() determine if the command succeeded. 
```suggestion
     * @return string[]|null
```

File: lib/Mage/System/Ftp.php [hunk 5]
```diff
@@ -308,7 +308,7 @@ public function close()
      *
      * @param $mode
      * @param $remoteFile
-     * @return bool
+     * @return int
```
Comment (kiatng) [inline lib/Mage/System/Ftp.php:311]: https://www.php.net/manual/en/function.ftp-chmod.php
> Returns the new file permissions on success or false on error. 
```suggestion
    * @return int|false
```

File: lib/Mage/System/Ftp.php [hunk 6]
```diff
@@ -359,7 +359,7 @@ public function get($localFile, $remoteFile, $fileMode = FTP_BINARY, $resumeOffs
      * ftp_nlist wrapper
      *
      * @param string $dir
-     * @return bool
+     * @return array
```
Comment (kiatng) [inline lib/Mage/System/Ftp.php:362]: https://www.php.net/manual/en/function.ftp-nlist.php
> Returns an array of filenames from the specified directory on success or false on error. 
```suggestion
     * @return string[]|false
```

File: lib/Mage/System/Ftp.php [hunk 7]
```diff
@@ -373,7 +373,7 @@ public function nlist($dir = "/")
      *
      * @param string $dir
      * @param bool $recursive
-     * @return mixed
+     * @return array
```
Comment (kiatng) [inline lib/Mage/System/Ftp.php:376]: https://www.php.net/manual/en/function.ftp-rawlist.php
> Returns an array where each element corresponds to one line of text. Returns false when passed directory is invalid. 
```suggestion
     * @return string[]|false
```

File: lib/Varien/Filter/Array.php [hunk 8]
```diff
@@ -32,6 +32,7 @@ function addFilter(Zend_Filter_Interface $filter, $column='')
             }
             $this->_columnFilters[$column]->addFilter($filter);
         }
+        return $this;
```
Comment (kiatng) [inline lib/Varien/Filter/Array.php:35]: Add this for `addFileter()` and the following `filter`?
```php
    /**
     * @inheritdoc
     */
```

**Comment by sreichel** [comment]
Something went wrong. New PR later.

**Pull request closed without merging**